### ISSUE TITLE
BUG: Some text in light theme are not readable 

### ISSUE THREAD

**Issue by azmy60** [beekeeper-studio/beekeeper-studio#2620]
![image](https://github.com/user-attachments/assets/80def01a-fd0e-420e-84d9-0d4a79a64cd9)

Some text are white in light theme. Found this in status bar, new table, and edit table tab.

**Comment by azmy60** [comment]
Not only that. I get that because I switched from dark theme to light theme. And then nothing changes when I want to switch back to dark theme.

**Comment by rathboma** [comment]
Is this in 5 beta? Def a blocker if so. Can we get this fixed and PRd?

**Comment by rathboma** [comment]
This doesn't happen for me on master. More details?

**Comment by azmy60** [comment]
Ok. Maybe this is just in `monorepo-2024` branch. Idk what causes it tho.

**Comment by not-night-but** [comment]
@azmy60 I'm assuming this isn't an issue anymore?

**Comment by azmy60** [comment]
No. I guess it was an issue with the CSS that Bks UI Kit built at the time I was working on it.
